app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -61,7 +61,8 @@ miqHttpInject(angular.module('containerDashboard', ['ui.bootstrap', 'patternfly'
           id = '';
         }
         else {
-          id = '/'+ (/\/(\d+)/.exec($location.absUrl())[1]);
+          // search for http://<url>/<controller>/<number>...
+          id = '/' + (/\/.+\/.+\/(\d+)/.exec($location.absUrl())[1]);
```

Comment:
@himdel i think using the pattern you suggested [^\/] will fix it.
anchor is problematic because `$location.path()` does not work for me :-(
 
```
>> /:\/\/[^\/]+\/[^\/]+\/(\d+)/.exec('http://whatever/controller/1234?param=/456')
Array [ "://whatever/controller/1234", "1234" ]
>> /:\/\/[^\/]+\/[^\/]+\/(\d+)/.exec('http://10.5.6.6/controller/1234?param=/456')
Array [ "://10.5.6.6/controller/1234", "1234" ]
```